godot/Cargo.toml

```diff
@@ -8,7 +8,7 @@ keywords = ["gamedev", "godot", "engine", "2d", "3d"] # possibly: "ffi"
 categories = ["game-engines", "graphics"]
 
 [features]
-default = ["codegen-full"]
+default = ["codegen-full", "api-custom"]
```

Comment:
why?

Reply:
this uses a godot 4.3 feature. ill remove that when 4.3 is stable or if i add the #[api_since] things.

Reply:
you should add `#[since_api]` regardless, otherwise gdext would fail to compile for other api versions. but if this is just a development convenience for now then that's fair.